_templates/choose-your-wallet.html

```diff
@@ -105,6 +108,7 @@
           transparency: "checkpasstransparencyopensource"
           environment: "checkfailenvironmentdesktop"
           privacy: "checkgoodprivacyimproved"
+          fees: "checkpassfeecontroloverride"
```

Comment:
Armory's next version (which will be released soon) will contain CPFP and RBF. Is that good enough to have it listed as full control now or should I wait and PR that after the release?